packages/grafana-ui/src/components/DateTimePickers/DateTimePicker/DateTimePicker.tsx

```diff
@@ -261,6 +268,10 @@ const getStyles = (theme: GrafanaTheme2) => ({
     border-radius: ${theme.shape.borderRadius(1)};
     background-color: ${theme.colors.background.primary};
     z-index: ${theme.zIndex.modal};
+
+    button:disabled {
+      color: ${theme.colors.action.disabledText};
+    }
```

Comment:
This might not be specific enough, in case we have other buttons inside this container. We can put it [here](https://github.com/grafana/grafana/blob/89df757f95af49712b99a046ec5a9c5c300e92bf/packages/grafana-ui/src/components/DateTimePickers/TimeRangePicker/CalendarBody.tsx#L81) as just:

```
&:disabled {
  color: ${theme.colors.action.disabledText};
}
```

Reply:
You are right, missed that we had those specific tile styles, thank you! and fixed 🚀